Library/.rubocop.yml

```diff
@@ -6,6 +6,12 @@ AllCops:
     - '**/Casks/**/*'
     - '**/vendor/**/*'
 
+require:
+  - ./Homebrew/rubocops/bottle_block_cop.rb
```

Comment:
Yes, I was thinking of creating another another ```.yml``` file(which will have all custom cops) and```require``` it inside ```.rubocop.yml``` as the custom cops grow in number. Your idea of ```require``` on ```rubocops.rb``` is seems much better, but then we'd have to place all the cops in a single huge ```rubocops.rb``` rather than having individual files. If thats not a problem, I'll go ahead and change it.